api/grpcserver/mesh_service.go

```diff
@@ -106,20 +106,24 @@ func (s MeshService) MaxTransactionsPerSecond(context.Context, *pb.MaxTransactio
 
 // QUERIES
 
-func (s MeshService) getFilteredTransactions(startLayer types.LayerID, addr types.Address) (txs []*types.MeshTransaction, err error) {
+func (s MeshService) getFilteredTransactions(startLayer types.LayerID, addr types.Address) ([]*types.MeshTransaction, error) {
 	meshTxIds := s.getTxIdsFromMesh(startLayer, addr)
-	mempoolTxIds := s.Mempool.GetTxIdsByAddress(addr)
+	mempoolTxs := s.Mempool.GetTxsByAddress(addr)
```

Comment:
according to the conversation on this PR it seems like this should be removed entirely